README.md

```diff
@@ -67,6 +67,13 @@ terraform taint module.instances[\"us-west-2d\"].aws_instance.polygon_edge_insta
 terraform apply
 ```
 
+### Deployment
+
+VPC must be created beforehand, as the instances are deployed using a loop which must know all
+AZ names in advance.
+
+To do this, simply run `terraform apply -target=module.vpc` and then `terraform apply` after it.
```

Comment:
Lets try to avoid a targetted apply by fetching the AZ list ourselves

```terraform
data "aws_availability_zones" "current" {}

locals {
  azs = slice(data.aws_availability_zones.current.names, 0, 4)

} 
```